docs/sphinx/user_guide/feature/sort.rst

```diff
@@ -0,0 +1,141 @@
+.. ##
+.. ## Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+.. ## and other RAJA project contributors. See the RAJA/COPYRIGHT file
+.. ## for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+.. ##
+
+.. _sort-label:
+
+================
+Sorts
+================
+
+RAJA provides portable parallel sort operations, which are basic
+parallel algorithm building blocks. They are described in this section.
+
+A few important notes:
+
+.. note:: * All RAJA sort operations are in the namespace ``RAJA``.
+          * Each RAJA sort operation is a template on an *execution policy*
+            parameter. The same policy types used for ``RAJA::forall`` methods
+            may be used for RAJA sorts.
+          * RAJA sort operations accept an optional *comparator* argument so
+            users can perform different types of sort operations. If
+            no operator is given, the default is a 'less' operation and
+            the result is **non-decreasing**.
+
+Also:
+
+.. note:: * For sorts using the CUDA back-end, RAJA uses the implementations
+            provided by the NVIDIA cub library, which is available in the
```

Comment:
Would it be better to just cross-reference the instructions for using an external cub on this page: https://raja.readthedocs.io/en/v0.11.0/getting_started.html? That way we don't have to modify multiple doc locations if things change.